ignite/contrib/handlers/visdom_logger.py

```diff
@@ -0,0 +1,305 @@
+from ignite.engine import Engine
+from ignite.engine import Events
+import os
+import numpy as np
+from typing import Callable, Dict, List, Union
+
+
+class _VisdomWindows:
+    """Object encapsulating a Visdom Window.
+
+    Args:
+        vis (Visdom object): Visdom client.
+        env (str): Name of Visdom environment for the graphs.
+        save_by_default (bool): The graphs will be saved by default by the server.
```

Comment:
This option name is not clear. According to the API https://github.com/facebookresearch/visdom#vissave
this saves some environment of the server. Maybe we could leave it by default as true ?

Reply:
This option saves the environment (your current plots) on the server side. Without this option if the server is closed (crashes) and opened again, all past work is lost.
I agree that it is better to have it True by default.